### DISCUSSION TITLE
Please guide - Simple way to access WordPress website on same LAN / Wifi

### DISCUSSION THREAD

**Discussion by KoolPal** [leokhoa/laragon#765] [Q&A]
Hi,

I am trying to launch my local WordPress site https://pb.test/ on my **mobile** which is on the same WiFi as my laptop with Laragon

Please note:
- I am not looking for ngrok solution. Local LAN only, please
- I am looking at using HTTPS too for testing my payment gateway whose sandbox requires HTTPS

I have been searching for days and I am stuck at on my my **mobile** which is on the same WiFi as my laptop with Laragon
https://192.168.29.104/pb.test/ shows this
![image](https://github.com/leokhoa/laragon/assets/454883/1ea9f8c6-f6c2-4db1-81bf-727ace54c924)

This is my current setup
![image](https://github.com/leokhoa/laragon/assets/454883/523cb090-3fd7-4a4f-99f6-c93f9944aa13)

I have tried to figure the steps here https://github.com/leokhoa/laragon/discussions/476#discussioncomment-5896049 

**Changes attempted**
Changed **deny all;**, to **allow        all;**
at
S:\laragon\etc\nginx\sites-enabled\00-default.conf
S:\laragon\usr\tpl\nginx.sites-enabled.00-default.manifest.tpl
S:\laragon\usr\tpl\nginx.sites-enabled.00-default.SSL.manifest.tpl
**Restarted Laragon.** 
But cannot move beyond 
https://192.168.29.104/pb.test/ shows this
![image](https://github.com/leokhoa/laragon/assets/454883/1ea9f8c6-f6c2-4db1-81bf-727ace54c924)

What am I missing?

Please guide with baby steps. I am not a developer.

Thanks a lot!

**Comment by Pen-y-Fan** [comment]
Click the cog in the top right of Laragon and try changing the document root to S:\laragon\www\pd

<img width="400" alt="image" src="https://github.com/leokhoa/laragon/assets/40126936/1b48acf5-9fd1-45ff-aa6c-9843e9ba3846">

Once you have accepted all the Windows security elevated privileges requests. You should now be able to access the site from your mobile using http://192.168.29.104/
